src/vehicle_part.cpp

```diff
@@ -136,6 +136,11 @@ std::string vehicle_part::name( bool with_prefix ) const
             break;
         }
     }
+    // Update the vehicle.cpp line 6127 when you change this %
```

Comment:
This should be defined as a constant in the header if it's used in multiple places

Reply:
Thanks commited it 👍